daft/dataframe/dataframe.py

```diff
@@ -2035,7 +2011,13 @@ def agg(self, *to_agg: Union[Expression, Iterable[Expression]]) -> "DataFrame":
         Returns:
             DataFrame: DataFrame with aggregated results
         """
-        return self._agg(self._agg_inputs_to_expressions(to_agg), group_by=None)
```

Comment:
I know some folks will definitely struggle with this change -- any chance we could detect tuples as a validation step and throw a ValueError here indicating that they should use the new API since it was deprecated in 0.3?

Reply:
Yeah I can add that